src/Eto/Forms/ThemedControls/ThemedDocumentControlHandler.cs

```diff
@@ -215,9 +216,9 @@ public Color TabForegroundColor
 	}
 
 	/// <summary>
-	/// Gets or sets the highlight foreground color for the close button.
+	/// Gets or sets the highlight foreground color for the highlighted (or active) tab.
```

Comment:
same as this [comment](https://github.com/picoe/Eto/pull/2605#discussion_r1452046713)

Reply:
Updated in [2be82e2](https://github.com/picoe/Eto/pull/2605/commits/2be82e215805ebc1d82d71b7a07b39d655f14996).